packages/core-api/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@backstage/core-api",
+  "name": "@zalando-sunrise/backstage-core-api",
```

Comment:
Isn't it called `core` without "api"?
https://www.npmjs.com/package/@zalando-sunrise/backstage-core

Reply:
There are `backstage/core` and  [backstage/core-api](https://www.npmjs.com/package/@backstage/core-api).